force-app/main/default/classes/AccountTriggerHandler.cls

```diff
@@ -0,0 +1,84 @@
+public class AccountTriggerHandler extends TriggerHandler {
+
+    private List<Account> newAcctList;
+    private List<Account> oldAcctList;
+    private Map<Id,Account> newAcctMap;
+    private Map<Id,Account> oldAcctMap;
+
+    public AccountTriggerHandler() {
+        
+        this.newAcctList = (List<Account>)Trigger.new;
+        this.oldAcctList = (List<Account>)Trigger.old;
+        this.newAcctMap = (Map<Id,Account>)Trigger.newMap;
+        this.oldAcctMap = (Map<Id,Account>)Trigger.oldMap;
+        //this.setMaxLoopCount(1);
```

Comment:
Just a random thought that I don't think I have ever used Trigger.newMap or Trigger.oldList in my career. :)